hw_zolotikov/calculator.py

```diff
@@ -0,0 +1,27 @@
+def mymult(x, y):
+        return x * y
+def mysub(x, y):
+        return x - y
+def mysum(a, b):
+    return a+b
+def mydiv(x,y):
+    return x/y
+def main():
+    inp = input('vvedite to, chto nado poschitat')
+    inp = inp.split(sep =' ')
+    x = float(inp[0])
+    y = float(inp[2])
+    sign = inp[1]
```

Comment:
В целом код рабочий, хоть и не очень питонячий. Это ничего страшного, мы с вами еще научимся. Например, тут это можно было бы сделать так:
```suggestion
    num1, operator, num2 = input().split()
    num1, num2 = float(num1), float(num2)
```

Название sign в целом не плохое, но я не уверен что в английском "знак" в смысле математического знака так и переводится. Поэтому я бы сделал operator, хоть так и подлиннее.